backend/packages/Upgrade/src/api/controllers/validators/MarkExperimentValidator.v4.ts

```diff
@@ -0,0 +1,27 @@
+import { IsNotEmpty, IsDefined } from 'class-validator';
+import { MARKED_DECISION_POINT_STATUS } from 'upgrade_types';
+
+export class MarkExperimentValidator2 {
+  @IsNotEmpty()
+  @IsDefined()
+  public userId: string;
+
+  public data: {
+    site: string;
+    target: string | undefined;
+    assignedCondition: { conditionCode: string; experimentId: string };
+    assignedFactor: object | undefined;
+  };
+
+  @IsNotEmpty()
+  @IsDefined()
+  public site: string;
+
+  @IsNotEmpty()
+  @IsDefined()
+  public assignedCondition: { conditionCode: string; experimentId: string };
```

Comment:
Similarly here, I believe we only use `data.assignedCondition` and `data.assignedFactor` so can we remove these two?

Reply:
Same as above